lib/env-validator.ts

```
@@ -0,0 +1,116 @@
+    ]
+    
+    for (const varName of required) {
+      if (!process.env[varName]) {
+        this.errors.push(`Missing required environment variable: ${varName}`)
+      }
+    }
+    
+    // Validate formats
+    if (process.env.CLOUDFLARE_ACCOUNT_ID && 
+        !/^[a-f0-9]{32}$/.test(process.env.CLOUDFLARE_ACCOUNT_ID)) {
+      this.errors.push('CLOUDFLARE_ACCOUNT_ID must be a 32-character hex string')
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)